docs/official_devices/list.md

```diff
@@ -27,6 +27,7 @@ sidebar_class_name: hidden
 ### Xiaomi
 - [POCO F3 / Mi 11x / Redmi K40 (alioth)](/devices/alioth)
 - [POCO M3 Pro/Redmi Note 10T/10 5G/11 SE (camellia)](/devices/camellia)
+- [POCO x5 5g / Redmi note 12 5g (stone)](/devices/stone)
```

Comment:
Why are you adding the device to discontinued support?
Also, please use proper naming (capitalization, etc.)

Here's my suggestion:
```
- [POCO X5 5G / Redmi Note 12 5G(stone)](/devices/stone)
```